api/src/pcapi/core/offerers/api.py

```diff
@@ -91,6 +91,7 @@ def update_venue(
     opening_days: list[serialize_base.OpeningHoursModel] | None = None,
     contact_data: serialize_base.VenueContactModel | None = None,
     criteria: list[criteria_models.Criterion] | offerers_constants.T_UNCHANGED = offerers_constants.UNCHANGED,
+    external_accessibility_url: str | offerers_constants.T_UNCHANGED = offerers_constants.UNCHANGED,
```

Comment:
Si on peut effacer l'adresse :
```suggestion
    external_accessibility_url: str | None | offerers_constants.T_UNCHANGED = offerers_constants.UNCHANGED,
```